src/hardware/vga_other.cpp

```diff
@@ -616,20 +616,14 @@ static void PCJr_FindMode()
 			} else {
 				VGA_SetMode(M_TANDY2);
 			}
-		} else {
-			/* otherwise some 4-colour graphics mode */
+		}
+		/* otherwise some 4-colour graphics mode */
+		else {
+			const auto new_mode = (cga_comp == 1) ? M_CGA16 : M_TANDY4;
```

Comment:
Nice simplification :)

Reply:
Thanks :+1: I knew you'd appreciated it!
Collapsing two levels of nested if/else felt good.